File: src/main/java/com/haozileung/web/service/permission/IPermissionService.java
package com.haozileung.web.service.permission;

import com.haozileung.infra.pager.PageResult;
import com.haozileung.web.domain.permission.Permission;


public interface IPermissionService {

    void save(Permission dictionary);

    void update(Permission dictionary);

    void delete(Permission dictionary);

    PageResult<Permission> query(Permission dictionary, int pageNo, int pageSize);
}

File: src/main/kotlin/com/haozileung/web/service/permission/IPermissionService.kt
package com.haozileung.web.service.permission

import com.haozileung.infra.page.PageResult
import com.haozileung.web.domain.permission.Permission


interface IPermissionService {

    fun save(dictionary: Permission)

    fun update(dictionary: Permission)

    fun delete(dictionary: Permission)

    fun query(dictionary: Permission, pageNo: Int, pageSize: Int): PageResult<Permission>
}
